mlrun/model_monitoring/controller.py

```
@@ -55,29 +68,32 @@ def __init__(
         self._endpoint = endpoint
         self._application = application
         self._first_request = first_request
-        self._kv_storage = get_v3io_client(endpoint=mlrun.mlconf.v3io_api).kv
+        self._kv_storage = get_v3io_client(
+            endpoint=mlrun.mlconf.v3io_api, logger=logger
+        ).kv
         self._v3io_container = self.V3IO_CONTAINER_FORMAT.format(project=project)
         self._start = self._get_last_analyzed()
         self._stop = last_updated
         self._step = timedelta_seconds
 
     def _get_last_analyzed(self) -> Optional[int]:
         try:
-            data = self._kv_storage.get(
-                container=self._v3io_container,
-                table_path=self._endpoint,
-                key=self._application,
-            )
+            with _adapt_logger_level(logger=logger, level=logging.ERROR):
+                data = self._kv_storage.get(
+                    container=self._v3io_container,
+                    table_path=self._endpoint,
+                    key=self._application,
+                )
         except HttpResponseError as err:
-            logger.warn(
-                "Failed to get the last analyzed time for this endpoint and application, "
+            logger.info(
```

I think we should have app invocation counter. this way log an error if it's not the first invocation, and silently ignore this error (since it's expected) on the first invocation.

---

Created [ML-5496](https://jira.iguazeng.com/browse/ML-5496) following our discussion